### PR TITLE
L1 m0 repackaging with starter code for processing apps

### PR DESCRIPTION
**Pull request by jaLeague**
Please synch these changes with the parallel changes to the curriculum pages reflecting the same.

### CODE REVIEW

**Review by ColbySchexnayder — CHANGES_REQUESTED** [review]
It looks good but it looks like the answers were left in DuelingButtons.java and a small typo in the package name. The package is duelling but the recipe is dueling

**Review by dencee — CHANGES_REQUESTED** [review]
Fortune folder is misspelled
Level1-Module0/src/_03_gui_from_scratch/_1_forunte_cookie/recipe.html

**Comment by dencee** [comment]
What are your plans with the simple collision demo?

File: Level1-Module0/src/_02_gui_with_help/_5_duelling_buttons/DuelingButtons.java [hunk 1]
```diff
@@ -27,25 +27,26 @@
 	public void run() {
 
 		// 1. Add the panel to the frame
-
+frame.add(panel);
```
[the remark this answers is not in the document]
Comment (jaLeague) [reply Level1-Module0/src/_02_gui_with_help/_5_duelling_buttons/DuelingButtons.java]: Removed solution

File: Level1-Module0/src/_02_gui_with_help/_3_drum_kit/DrumKit.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+			Icon icon = new ImageIcon(imageURL);
+			imageLabel.setIcon(icon);
+		}
+		return imageLabel;
+	}
+
+
+	public static synchronized void playSound(String fileName)
+    {
+		if (canPlaySounds) {
```
[the remark this answers is not in the document]
Comment (jaLeague) [reply Level1-Module0/src/_02_gui_with_help/_3_drum_kit/DrumKit.java:95]: File should now be consistent

**Comment by jaLeague** [comment]
> It looks good but it looks like the answers were left in DuelingButtons.java and a small typo in the package name. The package is duelling but the recipe is dueling

Classes now consistent with package

**Comment by jaLeague** [comment]
I believe this is now ready for merging.

**Pull request merged**